code/game/machinery/computer/card.dm

```diff
@@ -39,6 +39,7 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		"Security Sergeant", //SKYRAT EDIT ADDITION - SEC_HAUL
 		"Blueshield",	//SKYRAT EDIT: Blueshield slots should never be above 1.
 		"Nanotrasen Representative", //SKYRAT EDIT ADDITION
+		"Command Secretary",	//SKYRAT EDIT: CommSec slots should never be above 1.
```

Comment:
```suggestion
		"Command Secretary", //SKYRAT EDIT: CommSec slots should never be above 1.
```
Don't use tab indents for comments, maybe fix the above ones too while you're here